cc/model/buffered_model.cc

```diff
@@ -17,18 +17,19 @@
 namespace minigo {
 
 BufferedModel::BufferedModel(std::vector<std::unique_ptr<Model>> impls)
-    : Model(impls[0]->name(), impls[0]->feature_type(),
+    : Model(impls[0]->name(), impls[0]->feature_descriptor(),
             static_cast<int>(impls.size())) {
   for (auto& x : impls) {
     // Make sure all impls use the same name & input features.
     MG_CHECK(x->name() == name());
-    MG_CHECK(x->feature_type() == feature_type());
+    MG_CHECK(x->feature_descriptor().set_bytes ==
```

Comment:
why is it just set_bytes that gets compared?

Reply:
I was being lazy :P